private/buf/bufgen/bufgen.go

```diff
@@ -44,8 +30,16 @@ const (
 	V1Version = "v1"
 	// V1Beta1Version is the string used to identify the v1beta1 version of the generate template.
 	V1Beta1Version = "v1beta1"
+	// V2Version is the string used to identify the v2 version of the generate template.
+	V2Version = "v2"
```

Comment:
Wondering if we should launch with a `v2alpha1` or `v2beta1` first, and then later support an stable v2.

Reply:
That is good point. @bufdev has confirmed before that it is fine to go with `v2` directly